chat/EurLexChat.py

```diff
@@ -372,52 +365,4 @@ def get_answer(self,
                 return Answer(answer=result.answer, new_documents=new_docs)
             else:
                 return Answer(answer=result.answer)
-        return Answer(answer=result.content)
-
-
-class EurLexChatAkn(EurLexChat):
-    def _parse_documents(self, docs: List[Document]) -> List[dict]:
-        """
-        Parse a list of documents into a standardized format.
-
-        Args:
-            docs (List[Document]): A list of documents to parse.
-
-        Returns:
-            List[dict]: A list of dictionaries, each containing parsed information from the input documents.
-        """
-
-        parsed_documents = []
-
-        for doc in docs:
-            parsed_documents.append({
-                'text': doc.page_content,
-                'source': doc.metadata["uri"],
-                '_id': doc.metadata["uri"] + doc.metadata["article_id"]
-            })
-        return parsed_documents
-
-    def get_relevant_docs(self, question: str, eurovoc: str = None) -> List[dict]:
-        """
-        Retrieve relevant documents based on a given question.
-
-        Args:
-            question (str): The question for which relevant documents are retrieved.
-            eurovoc (str): The Eurovoc to be used as filter
-
-        Returns:
-            List[dict]: A list of relevant documents.
-        """
-        if eurovoc:
-            retriever = get_vectorDB_module(
-                self.config['vectorDB'], self.embedder, metadata={
-                    'filter': {'eurovoc': ''}}
-            )
-            relevant_documents_pipeline_with_filter = (
-                retriever | self._parse_documents)
-            docs = relevant_documents_pipeline_with_filter.invoke(
-                question
-            )
-        else:
-            docs = self.relevant_documents_pipeline.invoke(question)
-        return docs
```

Comment:
Are we sure we don't want to use this code anymore?

Reply:
Well, I'm removing it because we are not using this class, moreover currently as it is the filter for eurovoc does not work so the class would have to be rewritten in any case. Do you think it's better to fix the code now and keep it in case we might need it in the future?

Reply:
no it's fine, but if we could keep it on a separate branch maybe that would be useful in case we wanted to reuse it what do you think?